src/shogun/clustering/GMM.cpp

```diff
@@ -423,7 +420,11 @@ void CGMM::partial_em(int32_t comp1, int32_t comp2, int32_t comp3, float64_t min
 		SGMatrix<float64_t> c2=components[2]->get_cov();
 		linalg::add(c1, c2, c1, alpha1, alpha2);
 
-		components[1]->set_d(SGVector<float64_t>(SGMatrix<float64_t>::compute_eigenvectors(c1.matrix, dim_n, dim_n), dim_n));
+		SGVector<float64_t> eigenvalues(dim_n);
```

Comment:
good!
Maybe we should also offer an API for just eigenvalues?
But that could be done later